blueocean-core-js/src/js/i18n/i18n.js

```diff
@@ -136,7 +137,13 @@ export default function i18nTranslator(pluginName, namespace) {
     const I18n = pluginI18next(pluginName, namespace);
 
     // Create and cache the translator instance.
-    translator = I18n.getFixedT(defaultLngDetector.detect(), namespace);
+    let detectedLang;
```

Comment:
so this isn't a functional change so much as making sure it falls back correctly?